lib/internal/util/inspect.js

```diff
@@ -792,6 +805,46 @@ function formatRaw(ctx, value, recurseTimes, typedArray) {
   return res;
 }
 
+function groupArrayElements(ctx, output) {
+  let totalLength = 0;
+  const maxLength = output.reduce((max, cur) => {
```

Comment:
Why use this (and the other instances) instead of a normal `for`-loop, especially since these `util` methods have seen many optimization PRs in recent times?

Reply:
I'll change it to simple for loops.

Please note though that V8 did a great job in improving idiomatic code (and I count lambda functions as such). I just wrote a tiny benchmark and the results look promising (I checked V8 7.0, 7.1 and 7.4).